### PR TITLE
Lc1658

### PR DESCRIPTION
**Pull request by snowan**

### CODE REVIEW

**Comment by snowan** [comment]
@sunboman review

**Comment by snowan** [comment]
@sunboman review

File: 2021/python/1658.minimum-operations-2-reduce-x-2-zero.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+sum == sum(array) - x.
+now the question becomes to find max subarray sum = target problem.
+use map(val, index) + two pointers
+"""
+def minOperations(nums: List[int], x: int) -> int:
+        target = sum(nums) - x
+        if target == 0:
+            return len(nums)
+        size = len(nums)
+        max_subarr = -1
```
Comment (sunboman) [inline 2021/python/1658.minimum-operations-2-reduce-x-2-zero.py]: `max_subarr, pre_sum = -1, 0`

**Review by snowan — COMMENTED** [review]
@sunboman nice~ fixed

**Review by sunboman — APPROVED** [review]

**Pull request merged**